plutus-use-cases/src/Language/PlutusTx/Coordination/Contracts/Game.hs

```diff
@@ -7,6 +7,9 @@
 {-# LANGUAGE DeriveAnyClass #-}
 {-# LANGUAGE DerivingStrategies #-}
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
huh? language extensions added but code not changed?

Reply:
Yeah, I think this comes from the fact that we didn't used to compile this file. We only compiled it by running it through the interpreter, which adds the flags on the way.

Now we have to compile it standalone it's caught those missing pragma.

tdlr; They should have been there. We're catching more bugs. Yay.